x-pack/plugins/apm/ftr_e2e/ftr_config_run.ts

```diff
@@ -23,7 +23,7 @@ async function testRunner({ getService }: FtrProviderContext) {
   const result = await cypressStart(getService, cypress.run);
 
   if (result && (result.status === 'failed' || result.totalFailed > 0)) {
-    throw new Error(`APM Cypress tests failed`);
+    process.exit(1);
```

Comment:
Is there a way to print useful information to the console? or is it already printed by cypress itself?

Reply:
Cypress already prints the list of tests both the ones that failed and the ones the were successful.